data/json/items/gun/223.json

```diff
@@ -151,138 +149,82 @@
     "weight": 2500,
     "volume": 6,
     "price": 350000, "//" : "Current milspec, so Expensive.",
-    "bashing": 12,
-    "to_hit": -1,
     "material": ["steel", "plastic"],
     "symbol": "(",
     "color": "dark_gray",
     "skill": "rifle",
     "ammo": "223",
-    "ranged_damage": -2,
-    "range": -12,
-    "dispersion": 150,
```

Comment:
The description states it is rather unwieldy without conversions, but it has really good stats - perfect accuracy, huge range, good damage, great recoil management etc.